bsp/Inc/CAN.h

```diff
@@ -19,6 +19,7 @@
  *
  * How to Use This Driver:
  * Init to initialize the CAN peripheral
+    ** Define a HAL_CAN_MspGPIOInit function to init your GPIO pins, there is a default implementation in the driver
```

Comment:
Should the user be told to make HAL_CAN_MspGPIODeInit as well? I know we don't really expect them to deinit can, but just to be safe...